block/fork.go

```diff
@@ -0,0 +1,110 @@
+package block
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/dymensionxyz/dymint/node/events"
+	"github.com/dymensionxyz/dymint/types"
+	uevent "github.com/dymensionxyz/dymint/utils/event"
+)
+
+// MonitorForkUpdate listens to the hub
+func (m *Manager) MonitorForkUpdate(ctx context.Context) error {
```

Comment:
not sure if its done on purpose, but its MonitorForkUpdate not called when starting the manager